webapp/src/js/enketo/widgets/mrdt.js

```diff
@@ -33,7 +33,7 @@ define( function( require, exports, module ) {
     Mrdtwidget.prototype._init = function() {
         var self = this;
         var $el = $( this.element );
-        var $input = $el.find( 'input' );
+        var $input = $el.find( '.or-appearance-mrdt-image > input' );
```

Comment:
The name `$input` seems ambiguous now, since there are two inputs used by the widget. Maybe `$imageInput`?